server/leafnode.go

```diff
@@ -1057,6 +1065,10 @@ func (c *client) processLeafNodeConnect(s *Server, arg []byte, lang string) erro
 		c.leaf.isSpoke = true
 	}
 
+	// If we have permissions bound to this leafnode we need to send then back to the
+	// origin server for local enforcement.
+	s.sendPermsInfo(c)
```

Comment:
But LN permissions IIRC are only configured in the "remotes" section of leafnodes block, that is, only soliciting LNs configured. And if we process a CONNECT, we are not the soliciting, so there should not be any permissions here???

Reply:
We are sending back the perms that are associated with this user. This user was sent from the spoke but does not know what permissions are associated with the bound user on the hub.